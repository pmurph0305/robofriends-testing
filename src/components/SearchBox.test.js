import { shallow } from 'enzyme';
import React from 'react';
import SearchBox from './SearchBox';

it('expect to render SearchBox Component', () => {
    expect(shallow(<SearchBox/>)).toMatchSnapshot();
})