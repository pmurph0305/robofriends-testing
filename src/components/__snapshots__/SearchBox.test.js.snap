// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SearchBox Component 1`] = `ShallowWrapper {}`;
